### ISSUE TITLE
timeToPixels not returning the correct value for different zoom levels

### ISSUE THREAD

**Issue by jdelStrother** [bbc/peaks.js#203]
I'm going slightly crazy trying to figure out whether this broke with my recent changes or due to something else, and exactly what the correct fix is.   Was hoping you might be able to help...

If you try playback here - https://codepen.io/jdelStrother/pen/QOaVax?editors=1010 - and click the "toggle zoom" button, the playhead keeps jumping backwards momentarily every second or so.

As far as I can tell, this is because `WaveformZoomView.prototype.timeToPixels` is only returning the correct value when `this._zoomLevelIndex` is 0.

Thing is, I can't seem to reproduce this on the official peaks.js demo page.   Any suggestions what I'm screwing up?

**Comment by chrisn** [comment]
Curious. As you say, it's working OK in the demo page, but not in your example. I'll investigate.

**Comment by chrisn** [comment]
I can't reproduce the problem. I created this test page from your CodePen example, but it works as expected:
```html
<!DOCTYPE html>
<html>
  <head>
    <title>Peaks.js Demo Page</title>
    </style>
  </head>
  <body>
    <audio src="fish.mp3" controls></audio>
    <button id='timeupdate'>jump to t=10</button>
    <button id='zoom'>toggle zoom</button>
    <div id='logger'>loading...</div>
    <div id='peaks-container'></div>
    <script src="peaks.js"></script>
    <script>
      (function(Peaks){
        var audio = document.querySelector('audio')
        var timeupdateBtn = document.querySelector('#timeupdate')
        var zoomBtn = document.querySelector('#zoom')
        var logger = document.querySelector('#logger')

        var p = Peaks.init({
          container: document.getElementById('peaks-container'),
          mediaElement: audio,
          audioContext: new AudioContext(),
          zoomLevels: [512, 1024]
        })
        logger.innerHTML = 'init peaks (be patient...)'
        p.on('points.ready', function() {
          logger.innerHTML = 'loaded'
        })

        timeupdateBtn.addEventListener('click', function() {
          audio.currentTime = 10;
        })

        zoomBtn.addEventListener('click', function() {
          p.zoom.setZoom(p.zoom.getZoom() == 0 ? 1 : 0)
        })
      })(peaks);
    </script>
  </body>
</html>
```

**Comment by chrisn** [comment]
I think there was a problem when I published 0.9.3 to npm, and the latest fixes weren't included. I've just published 0.9.4 which should be all up to date. The difference between your setup and mine is that you're using the browserified version from wzrd.in and I'm using a local build from source.

**Comment by jdelStrother** [comment]
Ohh, I _thought_ the minified code didn't seem to make sense.

Thanks, looks good now.
